server/src/main/java/org/elasticsearch/action/admin/cluster/node/tasks/list/TransportListTasksAction.java

```diff
@@ -48,6 +65,7 @@ public TransportListTasksAction(ClusterService clusterService, TransportService
             TaskInfo::from,
             ThreadPool.Names.MANAGEMENT
         );
+        this.threadPool = threadPool;
```

Comment:
nit: both `ClusterService` and `TransportService` already contain references to the `ThreadPool`, we don't need another constructor param here.